### PR TITLE
[DD-296] Link last updated date to last commit on pages

### PR DESCRIPTION
**Pull request by rsylvian**
Ticket: [DD-296](https://circleci.atlassian.net/browse/DD-296) 
Preview: [link](http://circleci-doc-preview.s3-website-us-east-1.amazonaws.com/rs/DD-296/link-last-updated-to-github-with-preview/?force-all)
 
# Description
- Add a link to the page GitHub history on the Last updated date

# Reasons
Linking the last updated date to the last commit so the users can see what updates were made when the article was last updated. This should hopefully help users identify and be aware of what in the docs was/is outdated.

# Screenshots

Before:

image

After:

image

# Consideration
This will stay in a draft PR until https://github.com/circleci/circleci-docs/pull/6134 is not merged

### CODE REVIEW

File: jekyll/_layouts/classic-docs.html [hunk 1]
```diff
@@ -46,7 +46,12 @@
                   {% assign nowTimestamp = 'now' | date: '%s' %}  
                   {% assign timestampDiff = nowTimestamp | minus: dateStart %}
                   {% assign yearInSeconds = 60 | times: 60 | times: 24 | times: 365 %}
-                  {% if timestampDiff > yearInSeconds %} 1+ year ago {% else %} {{ page.last_modified_at | timeago }} {% endif %}
+                  {% assign ghLink = 'https://github.com/circleci/circleci-docs/commits/master/jekyll/' | append: page.path %}
```
Comment (nramabad) [inline jekyll/_layouts/classic-docs.html:49]: https://github.com/circleci/circleci-docs/blob/ac9317d8009b13191b348c01959a2978da65be19/jekyll/_config.yml#L10

Comment (rsylvian) [reply jekyll/_layouts/classic-docs.html:49]: Oh nice! Thanks!

File: Gemfile [hunk 2]
```diff
@@ -19,7 +19,8 @@ group :jekyll_plugins do
   gem 'jekyll-target-blank'
   gem 'jekyll-toc'
   gem 'jekyll-asciidoc'
-  gem "jekyll-last-modified-at"
+  gem 'jekyll-last-modified-at'
+  gem 'jekyll-timeago'
```
Comment (ccgeller) [inline Gemfile:23]: this should be in master now! :)

File: jekyll/_layouts/classic-docs.html (hunk 1 above)
Comment (ccgeller) [reply jekyll/_layouts/classic-docs.html:49]: This code has been moved to quick-info-list.html see [link](https://github.com/circleci/circleci-docs/pull/6077)

**Review by teesloane — APPROVED** [review]

**Pull request merged**